src/module/migration/runner/base.ts

```diff
@@ -14,7 +14,7 @@ interface CollectionDiff<T extends foundry.data.ActiveEffectSource | ItemSourceP
 export class MigrationRunnerBase {
     migrations: MigrationBase[];
 
-    static LATEST_SCHEMA_VERSION = 0.829;
+    static LATEST_SCHEMA_VERSION = 0.831;
```

Comment:
This going to break migrations for anyone running dev mode. Change this to 0.83 and the barbarian one to 0.831.

Reply:
So let's get the barbarian one merged first and I can fix the conflicts here.